src/Reflection/Adapter/ReflectionIntersectionType.php

```diff
@@ -18,13 +16,10 @@ public function __construct(private BetterReflectionIntersectionType $betterRefl
     {
     }
 
-    /** @return array<ReflectionNamedType> */
+    /** @return list<ReflectionUnionType|ReflectionIntersectionType|ReflectionNamedType> */
```

Comment:
This becomes a BC break that requires a new major 🤔

Reply:
Not valid anymore.